js/files_test.ts

```diff
@@ -18,3 +18,21 @@ test(async function filesCopyToStdout() {
   assertEqual(bytesWritten, fileSize);
   console.log("bytes written", bytesWritten);
 });
+
+test(async function createFileSuccess() {
+  const path = "test.txt";
+  let f = await deno.create(path);
```

Comment:
Could you create the test file in a temporary directory? Like using a path given by `deno.makeTempDir`

Reply:
Sure.

Reply:
Sure.